.vscode/settings.json

```diff
@@ -1,32 +1,21 @@
 {
     "editor.tabSize": 2,
-
-    "[javascript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.formatOnSave": true
-    },
-    "[typescript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode",
-        "editor.formatOnSave": true
-    },
-
     "eslint.alwaysShowStatus": true,
-    "eslint.autoFixOnSave": true,
     "eslint.lintTask.enable": true,
     "eslint.codeAction.showDocumentation": {
         "enable": true
     },
-    "eslint.validate": [
-        "javascript",
-        {
-            "autoFix": true,
-            "language": "typescript"
-        },
-    ],
     "editor.codeActionsOnSave": {
         "source.fixAll.eslint": true
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+    "eslint.workingDirectories": [
+        {
+            "directory": "./",
+            "changeProcessCWD": true
+        }
+    ],
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
It tells Visual Studio to default to the typescript version included in the project rather than your setting in the editor / workspace. I was working in a workspace with both this and the updated version xpring-common-js and they had two different versions of TS which was causing issues in my editor. (Now both are on 3.7.5)